tasks/db_deploy/migrate_db.py

```diff
@@ -22,27 +22,27 @@ def perform_migration(current_schema_version: int, config: Dict[str, str]) -> No
     """
     # Determine migrations to run based on current_schema_version and
     # update the versions table based on the latest_schema_version.
-    if current_schema_version == 1:
```

Comment:
What happens if the current_schema_version IS 1?

Reply:
yes it will not do anything right now. Should add an else statement.

If we decide to keep v1, then will add that code.

Reply:
added to if else statement